packages/dso-toolkit/components/Voorbeelden/Homepage.njk

```diff
@@ -1,83 +1,211 @@
+
+<style>
```

Comment:
Deze styling moet ergens anders. Deze voorbeeldpagina kan niet zonder de styling over te nemen worden nagemaakt. En we willen dat afnemers geen styling overnemen.

Reply:
Ik heb de styling naar een tile grid in een dso highlightbox (vraag is of dit de juiste plek is.)
Ik ben sowieso niet blij met deze oplossing omdat ik nu zeg voor een highlightbox is het altijd 3 tiles op een rij tenzij het een small screen is. Maar bij een ander ontwerp kan iemand anders weer maar 2 tiles per rij op een highlightbox willen. 

Eigenlijk zou ik dat kunnen in willen stellen hoeveel tiles ik wil per ontwerp/instantie dat het voorkomt. Ik kan nog geen vuistregel vinden. 

Wat is jouw mening?

Ik denk dat we dit niet zo kunnen vastleggen. Maar custom instellen van tile in rows is een complex vraagstuk compleet buiten scope van dit issue.